pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -272,7 +272,7 @@ func (ctx *context) prepareAckConnectionRecord(payload []byte) (*ConnectionRecor
 	ack := &model.Ack{}
 	err := json.Unmarshal(payload, ack)
 	if err != nil {
-		return nil, err
```

Comment:
Should this be a wrapped error rather than %s?

Likewise below?

Reply:
yes updated